Src/MoneyFox.Ui/Views/Settings/AccountLiteViewModel.cs

```diff
@@ -0,0 +1,8 @@
+namespace MoneyFox.Ui.Views.Settings;
+public sealed record AccountLiteViewModel(string Name)
+{
+    public override string ToString()
```

Comment:
Can't you just use `ItemDisplayBinding` on the Picker instead of overriding ToString()?

Reply:
This class itself is not required as we are just using the Name of the account, so I have removed it.